third_party/rust/crates/crates.bzl

```diff
@@ -5,6 +5,7 @@ cargo-raze generated Bazel file.
 DO NOT EDIT! Replaced on runs of cargo-raze
 """
 
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")  # buildifier: disable=load
```

Comment:
interesting.... seems cargo raze puts this line here even if this rule isn't used.